src/pages/patterns/buttons.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Buttons
+description: >
+  Know why they're pushing your buttons.
+---
+
+## Overview
+
+Buttons are common and vital elements because they help make content interactive across the
+internet. They allow a user to take actions with a single click and are placed throughout a
+page or workflow. Buttons use visual signifiers such as shape, color, and size to show that
+a user can interact with them.
+
+## Interface
+
+**Primary buttons** are for the most important action in a given view. There is typically
+one call-to-action on a page or view. **Default** buttons are for secondary actions.
+**Basic buttons** are for repetitive or deemphasized actions in a given view.
+
+![Tone matrix map](abstract:patterns-buttons-type-usage.png)
```

Comment:
This image is grossly oversized.

Reply:
@austingreendev could the Abstract plugin be structured to receive `@XxY` syntax similar to something used by https://placeholder.com/? So `abstract:image.png@300x150` would be 300px width, 150px height. And `abstract:image.png@300` would be 300px width, auto height.

Reply:
I'm not sure if that information would be possible to provide from there. We're really only updating the `src` URL with that remark plugin at this time. All the sizing and other responsive customizations happen in `gatsby-remark-images`.

If we want to get specific on a per-image basis we may want to reference the Abstract asset directly in a page query and apply customizations to an `Img` element.